integrations/utils/commonUtils.js

```diff
@@ -116,7 +116,27 @@ function flattenJson(data) {
   return result;
 }
 
+/**
+ * Check whether the passed eventname is mapped in the config
+ * and return the mapped event name.
+ * @param {*} event
+ * @param {*} eventsHashmap
+ * @returns mappedEventName
+ */
+function setEventMappingFromConfig(event, eventsHashmap) {
+  // if the event name is mapped in the config, use the mapped name
+  // else use the original event name
+  const events = [];
+  if (eventsHashmap[event]) {
```

Comment:
I have checked in these destinations, but to avoid any potential misses going ahead if other destinations are onboarded with this change, so kept it here aswell